duckduckgo/iOS
# COVID-19 CTA on Home Screen

Task/Issue URL: https://app.asana.com/0/856498667320406/1168005929568002
Tech Design URL:
CC:

**Description**:
Added a COVID-19 info CTA on the Home Screen.

**Steps to test this PR**:

For each Home screen mode (top/centered with favorited turned on/off) test:
1. Tapping on CTA should launch a SERP search.
2. Dismissing.
3. Favorite-enabled scenarios:
 - No favorites defined.
 - One fav defined.
 - Many favorites defined (taking more space than height of the screen).
4. Various screen sizes:
 - iPhone SE
 - iPhone 11
 - iPad
5. Light/Dark themes.


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
